src/hooks/wallets/mpc/useMPCWallet.ts

```diff
@@ -125,12 +126,31 @@ export const useMPCWallet = (): MPCWalletHook => {
     }
   }
 
+  const exportPk = async (password: string): Promise<string | undefined> => {
+    if (!mpcCoreKit) {
+      throw new Error('MPC Core Kit is not initialized')
+    }
+    const securityQuestions = new SecurityQuestionRecovery(mpcCoreKit)
+
+    try {
+      if (securityQuestions.isEnabled()) {
+        // Only export PK if recovery works
+        await securityQuestions.recoverWithPassword(password)
+      }
+      const exportedPK = await mpcCoreKit?._UNSAFE_exportTssKey()
+      return exportedPK
+    } catch (err) {
+      // TODO: Throw error through sentry
```

Comment:
Related to my other comment, should we rethrow an error here in order to display it to the user in the UI?